harness/determined/exec/harness.py

```diff
@@ -82,6 +82,11 @@ def main(chief_ip: Optional[str]) -> int:
         # without horovod, and we can't load the right horovod until we know which Trial class the
         # user implemented.
         trial_class, controller_class = load.get_trial_and_controller_class(env.experiment_config)
+        if info.container_rank == 0:
+            try:
+                analytics.send_analytics("trial_loaded", analytics.get_trial_analytics(trial_class))
```

Comment:
The trial class name is unlikely to be useful, since that will be a user-provided name.

It might be more useful to add a `._framework_name` or something to `PyTorchTrial` and friends and use the `trial_class._framework_name` instead, perhaps.

Reply:
An example data point is:

```
analytics.track('cfe8cddd-a184-4727-b937-27835fb6e33d', 'trial_loaded', {
    'library_version': {
        'determined': '0.17.6-dev0',
        'pytorch_lightning': '1.3.5',
        'torch': '1.9.0+cpu'
    },
    'trial_ancestors': 'GANTrial,LightningAdapter,PyTorchTrial,Trial,object',
    'trial_name': 'GANTrial'
})
```

> It might be more useful to add a ._framework_name or something to PyTorchTrial and friends and use the trial_class._framework_name instead, perhaps.

The framework is already tracked in the `trial_ancestors`. I think this is the cleanest way without bothering using `_framework_name`.

`trial_name` helps us to estimate (not exactly track) how examples are used directly.